.qube.yml

```diff
@@ -0,0 +1,19 @@
+qube_version: '2.7.0-SNAPSHOT # <<QUBE_NO_BUMP>>'
+full_name: Jennifer Bödker
+email: jennifer.boedker@qbic.uni-tuebingen.de
+project_name: core-utils-lib
+project_short_description: core-utils-lib. A qube based .
+version: 1.7.2
```

Comment:
This is the same version as on master, we probably need to upgrade the version..